### PR TITLE
PN-9548 - Refactor tests with new notification creation methods

### PR DESCRIPTION
**Pull request by alessandrogelmi**

### CODE REVIEW

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java [hunk 1]
```diff
@@ -28,13 +28,12 @@ public class DownloadFileMittentePagoPATest {
     private Map<String, Object> datiNotifica = new HashMap<>();
     private DownloadFile downloadFile;
 
-
     @When("Nella pagina Piattaforma Notifiche si clicca sulla notifica restituita")
     public void clickNotificaRestituita() {
         logger.info("Si clicca sulla notifica restituita");
-
+        logger.info("GENERATED IUN: " + System.getProperty("IUN"));
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java:34]: insert /n after latest log

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NewNotifichePagoPATest.java [hunk 2]
```diff
@@ -35,17 +35,16 @@ public void creoUnaNotificaConUnDestinatarioEUnDocumento() throws RestNotificati
             ArrayList<Document> documents = new ArrayList<>();
             documents.add(new Document());
             NewNotification notification = new NewNotification(DataPopulation.generatePaProtocolNumber(), "Pagamento Rata IMU", recipients, documents, PhysicalCommunicationTypeEnum.AR_REGISTERED_LETTER, "123456A", NotificationFeePolicyEnum.FLAT_RATE);
-
             NewNotificationResponse response = restNotification.newNotificationWithOneRecipientAndDocument(notification);
 
             if (response != null) {
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NewNotifichePagoPATest.java:40]: do condition with responseCode

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java (hunk 1 above)
Comment (afus0) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java:34]: Each log has its own newline

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NewNotifichePagoPATest.java (hunk 2 above)
Comment (afus0) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NewNotifichePagoPATest.java:40]: Please take a look to the method `newNotificationWithOneRecipientAndDocument()` you'll notice what you asked

**Review by leleOFA — APPROVED** [review]

**Pull request merged**